docs/content/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md

```diff
@@ -45,8 +45,8 @@ You could also programmatically set it as follows:
 Configuration config = new Configuration();
 
 // register the class of the serializer as serializer for a type
-config.set(PipelineOptions.SERIALIZATION_CONFIG, 
-    "[org.example.MyCustomType: {type: kryo, kryo-type: registered, class: org.example.MyCustomSerializer}]");
+config.set(PipelineOptions.SERIALIZATION_CONFIG,
+    List.of("org.example.MyCustomType: {type: kryo, kryo-type: registered, class: org.example.MyCustomSerializer}"));
```

Comment:
I notice that the way it was is the way that you need to specify it in Scala. For Java the code actually has an example of ```
config.set(
                PipelineOptions.SERIALIZATION_CONFIG,
                Collections.singletonList(
                        String.format(
                                "%s: {type: kryo, kryo-type: default, class: %s}",
                                PackagedProgramUtilsPipelineTest.class.getName(),
                                userSerializerClassName)));
```
would it not be better to copy the existing code and use "Collections.singletonList("

Reply:
Hi, "List.of" is also a valid Java way and is not related to Scala.